bluetail/tests/test_scan_contracts_command.py

```diff
@@ -0,0 +1,13 @@
+from io import StringIO
+from django.test import TestCase
+from django.core.management import call_command
+
+
+class TestScanContractsCommand(TestCase):
+    def test_it_runs(self):
+        # Make sure we've got the flags etc loaded.
+        call_command('insert_example_data')
+
+        # Send stdout/err to this StringIO object so it doesn't clutter test output.
+        null = StringIO()
```

Comment:
FYI  I noticed the django logger seems to be hidden from test output:
https://github.com/mysociety/bluetail/blob/382a2527521e11c285a16801d7e6c94988c184aa/bluetail/management/commands/insert_example_data.py#L13
This also allows us to add timestamps to the output easily later if its helpful?